pkgs/by-name/ni/nix-plugin-pijul/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  cmake,
+  pkg-config,
+  boost,
+  howard-hinnant-date,
+  nix,
```

Comment:
Does not build with nix 2.24: https://hydra.nixos.org/build/272877644/nixlog/1

Reply:
also seems to fail on nix 2.18 with aarch64-linux.

Reply:
That build log seems darwin specific, I think. Which is weird because I'm using this on my ARM Mac. I'll take a look at both once I'm home from vacation.